### PR TITLE
feat(components/lists): auto-scroll window or parent when dragging repeater items

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**
[AB#2240371](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2240371)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1253/storybook/)

Component Storybooks:

- [a11y-storybook](https://blackbaud.github.io/skyux-pr-preview/1253/storybooks/a11y-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1253/storybooks/ag-grid-storybook/)
- [core-storybook](https://blackbaud.github.io/skyux-pr-preview/1253/storybooks/core-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/1253/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1253/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/1253/storybooks/lists-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/1253/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1253/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/1253/storybooks/popovers-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1253/storybooks/split-view-storybook/)
- [storybook](https://blackbaud.github.io/skyux-pr-preview/1253/storybooks/storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1253/storybooks/tabs-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/1253/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/1253/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1253/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1253/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1253/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1253/playground/)

File: libs/components/lists/src/lib/modules/repeater/repeater-auto-scroll.service.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+import { SkyRepeaterAutoScrollOptions } from './repeater-auto-scroll-options';
+import { SkyRepeaterAutoScroller } from './repeater-auto-scroller';
+
+/**
+ * @internal
+ */
+@Injectable()
+export class SkyRepeaterAutoScrollService {
+  autoScroll(
```
Comment (Blackbaud-SteveBrush) [inline libs/components/lists/src/lib/modules/repeater/repeater-auto-scroll.service.ts]: Needs `public`.

Comment (Blackbaud-PaulCrowder) [reply libs/components/lists/src/lib/modules/repeater/repeater-auto-scroll.service.ts]: Fixed

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Comment by blackbaud-sky-build-user** [comment]
Cherry-pick [failed](https://github.com/blackbaud/skyux/actions/runs/4681224040)! :x: Please resolve conflicts and create a new PR.

**Pull request merged**